docs/reference/commandline/node_update.md

```diff
@@ -25,6 +25,38 @@ Options:
       --role string           Role of the node (worker/manager)
 ```
 
+### Add a metadata to a node
```

Comment:
Wondering if we should mention "labels" in the heading as well, no real suggestion what the title should be :blush: